apps/golden-sample-app/project.json

```diff
@@ -23,7 +23,12 @@
         "tsConfig": "apps/golden-sample-app/tsconfig.app.json",
         "assets": [
           "apps/golden-sample-app/src/favicon.ico",
-          "apps/golden-sample-app/src/assets"
+          "apps/golden-sample-app/src/assets",
```

Comment:
@manisha-backbase Do you need to do this step every time you add a new journey with assets?

Reply:
Yes, this would be necessary to load the assets used in a Journey.